examples/kubernetes/windows/delete.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+kubectl delete -f $DIR/powershell-nettest-pod.yaml
+kubectl delete -f $DIR/nettest-pod.yaml
+kubectl delete -f $DIR/pvc.yaml
+kubectl delete -f $DIR/pv.yaml
+kubectl delete -f $DIR/secrets.yaml
+kubectl delete -f $DIR/csi-driver.yaml
```

Comment:
I don't think driver install/uninstall (which is typically a cluster bootstrapping operation) should go with k8s workload install/uninstall.  I think you will also have teardown issues if the csi driver ends up being terminated before the workload is.

Reply:
Done.